File: Data Structures & Algorithms/max-consecutive-ones/submission-1.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        maxC = 0
        curr = 0

        for num in nums:
            if num == 1:
                curr += 1
                if curr > maxC:
                    maxC = curr
            else:
                curr = 0

        return maxC
        

File: Data Structures & Algorithms/majority-element-ii/submission-1.cpp
class Solution {
public:
    vector<int> majorityElement(vector<int>& nums) {
        unordered_map<int, int> mp;

        for (int num : nums) {
            mp[num]++;
        }

        vector<int> res;

        for (auto& pair : mp) {
            if (pair.second > nums.size() / 3) {
                res.push_back(pair.first);
            }
        }

        return res;
        
    }
};